mockk/common/src/test/kotlin/io/mockk/gh/Issue614Test.kt

```diff
@@ -0,0 +1,22 @@
+package io.mockk.gh
+
+import io.mockk.mockk
+import io.mockk.verifyOrder
+import kotlin.test.Test
+import kotlin.test.assertFails
+import kotlin.test.assertSame
+
+class Issue614Test {
+
+    @Test
+    fun `verifyOrder should throw AssertionError if no matching is found`() {
+        val mock: Something = mockk(relaxed = true, relaxUnitFun = true)
+
+        val exception = assertFails { verifyOrder { mock.doSomething() } }
```

Comment:
I think you can write this as 
```
assertFailsWith<AssertionError> {
    verifyOrder { mock.doSomething() }
}
```

Which looks a bit better to me.

What do you think?

Reply:
Nice catch, I'll update it